index.js

```diff
@@ -73,7 +73,7 @@ function arg(opts, {argv = process.argv.slice(2), permissive = false, stopAtPosi
 
 			for (let j = 0; j < separatedArguments.length; j++) {
 				const arg = separatedArguments[j];
-				const [originalArgName, argStr] = arg[1] === '-' ? arg.split('=', 2) : [arg, undefined];
+				const [originalArgName, argStr] = arg[1] === '-' ? arg.split(new RegExp('=(.+)'), 2) : [arg, undefined];
```

Comment:
```suggestion
				const [originalArgName, argStr] = arg[1] === '-' ? arg.split(/=(.*)/, 2) : [arg, undefined];
```

Just use the regex syntax here.

Also, use `.*` since we need to guarantee two arguments are always returned. With `.+`, only one is returned if the argument is `--foo=`, which would throw an array destructuring error.

Reply:
@Qix- I have made the changes and pushed a new commit
- `.+` changed to `.*`
- Used `/[=](.*)/` instead of `/=(.*)/` 'cos of lint rule [no-div-regex](https://eslint.org/docs/rules/no-div-regex)

Reply:
See https://github.com/zeit/arg/pull/48#discussion_r302926360